### PR TITLE
Plots and datacard option

### PR DESCRIPTION
**Pull request by minerva1993**
-Change figure style
-histogram names are changed, reflected in gather/preparing code
-rMin -20 rMax 20

### CODE REVIEW

File: gather_th1_for_limits.py [hunk 1]
```diff
@@ -14,12 +15,13 @@
 #jet_strings = ['j3', 'j4']
 jet_strings = ['j3b2', 'j3b3', 'j4b2', 'j4b3', 'j4b4']
 #training_strings = ['01', '02', '03', '04']
-training_strings = ['01']
+training_strings = ['0101010101'] #ver of j3b2+j3b3+j4b2+j4b3+j4b4
```
Comment (BrieucF) [inline gather_th1_for_limits.py:18]: Could you comment a bit more? Do you have a training with all jet categories together?

File: prepareShapesAndCards.py [hunk 2]
```diff
@@ -529,20 +545,51 @@ def prepareShapes(backgrounds, signals, discriminant, discriminantName):
...
                     )
+            if options.dataYear == '2016':
+                cb.cp().AddSyst(cb, 'hdamp_2016', 'lnN', ch.SystMap('process')
+                        (['ttbb', 'ttcc', 'ttlf'], 1.05)
+                        )
+                cb.cp().AddSyst(cb, 'scale_2016', 'lnN', ch.SystMap('process')
+                        (['ttbb', 'ttcc', 'ttlf'], 1.15)
+                        )
+                for i in xrange(len(discriminant)):
+                    cb.cp().AddSyst(cb, '$PROCESS_norm_'+str(discriminant[i][1]), 'lnN', ch.SystMap('process')
```
Comment (BrieucF) [inline prepareShapesAndCards.py]: Does'nt this leads to norm uncertainty uncorrelated for all jet bins while it was one for j3* and one for j4*?

Comment (minerva1993) [reply prepareShapesAndCards.py]: Arr ... this is my mistake. I will correct it

File: gather_th1_for_limits.py (hunk 1 above)
Comment (minerva1993) [reply gather_th1_for_limits.py:18]: Trained separately, but histograms are drawn in one code. Here, 0101010101 means the version of BDT classifiers in order of j3b2, j3b3, j4b2, j4b3, j4b4.

**Pull request merged**